src/clf/create_data.py

```diff
@@ -9,9 +9,10 @@
 import os
 
 import pandas as pd
-from constants import RAW_DATA_DIR, X_CLASSIFIED, X_MASKED
 from sklearn.model_selection import train_test_split
 
+from src.clf.constants import RAW_DATA_DIR, X_CLASSIFIED, X_MASKED
```

Comment:
funktioniert nur mit src.clf das generieren der docs oder ist das einfach einheitlich zu anderen scripts?

Reply:
Du brauchst den setup.cfg und pyproject.toml file in deiner branch dafür. Und dann musst du `pip install -e .` laufen lassen